draft-ietf-rats-corim.md

```diff
@@ -1217,14 +1217,18 @@ matched.
 ~~~
 
 The series object is an array of `conditional-series-record` that has both Reference and Endorsed Values.
-Each `conditional-series-record` record is evaluated in the order it appears in the series array.
+
+Each `conditional-series-record` has measurements belonging to one or more measured elements, pertaining to the environment that appears in the stateful-environment record.
+
 The Endorsed Values are accepted if the series condition in a `conditional-series-record` matches the ACS.
-The first `conditional-series-record` that successfully matches an ACS Entry terminates the matching and the corresponding Endorsed Values are accepted.
+The first `conditional-series-record` that successfully matches an ACS Entry terminates the matching and the corresponding Endorsed Values are accepted. In order for a `conditional-series-record` to match, measurements pertaining to every measured element in the record MUST match.
+
 If none of the series conditions match an ACS Entry, the triple is not matched,
```

Comment:
```suggestion
If none of the `selection` values match in the chosen ACS entry, the triple is not matched,
```

Reply:
Are we expressing the semantics in terms of the triple or in terms of the relation of internal representations we translate triples to? I think we should be consistent and use the internal representation for defining semantics.